docker-compose-services/meilisearch/docker-compose.meilisearch.yaml

```diff
@@ -0,0 +1,27 @@
+version: '0.36'
```

Comment:
Should be something like `version: '3.6'`

Reply:
```suggestion
version: '3.6'
```

Reply:
has been changed to '3.6'. thanks for pointing out.